### PR TITLE
[Feature] Start a conference call

### PR DESCRIPTION
**Pull request by johnxnguyen**
## What's new in this PR?

This PR enables the app to start both audio and video conference calls. This feature is behind a new feature flag `CONFERENCE_CALLING`. The feature flag is `false` by default, and `true` only for the `dev`, `internal` and `experimental` build flavors.
### Testing

1. Make a build with one of the aforementioned flavors.
2. Start a call in a group conversation.
3. The call should connect to the SFT server, and you should be the only participant in the call.

## Notes

This feature flag is temporary and will be after the feature is officially launched.
#### APK
[Download build #2272](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2272/artifact/build/artifact/wire-dev-PR2903-2272.apk)
[Download build #2273](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2273/artifact/build/artifact/wire-dev-PR2903-2273.apk)

### CODE REVIEW

File: wire-android-sync-engine/zmessaging/src/main/scala/com/waz/service/call/CallingServiceImpl.scala [hunk 1]
```diff
@@ -423,7 +424,10 @@ class CallingServiceImpl(val accountId:       UserId,
           if (convSize > VideoCallMaxMembers) Avs.WCallType.ForcedAudio
           else if (isVideo) Avs.WCallType.Video
           else Avs.WCallType.Normal
-        convType = if (isGroup) Avs.WCallConvType.Group else Avs.WCallConvType.OneOnOne
+        convType =
+          if (isGroup && conferenceCallingEnabled) Avs.WCallConvType.Conference
```
Comment (mejdoo) [inline wire-android-sync-engine/zmessaging/src/main/scala/com/waz/service/call/CallingServiceImpl.scala:428]: 👍

File: app/build.gradle [hunk 2]
```diff
@@ -144,6 +144,9 @@ android {
         buildConfigField 'boolean', 'KOTLIN_SETTINGS', "$rootProject.ext.kotlinSettings"
         buildConfigField 'boolean', 'KOTLIN_REGISTRATION', "$rootProject.ext.kotlinRegistration"
         buildConfigField 'boolean', 'KOTLIN_CORE', "$rootProject.ext.kotlinCore"
+
+        // Internal feature flags
+        buildConfigField 'boolean', 'CONFERENCE_CALLING', "$rootProject.ext.conferenceCalling"
```
[the remark this answers is not in the document]
Comment (gizemb) [reply app/build.gradle:149]: we don't actually need to put this value into rootProject.ext if we only need it from app module

[the remark this answers is not in the document]
Comment (johnxnguyen) [reply app/build.gradle:149]: Should I just set it to `false` here instead?

[the remark this answers is not in the document]
Comment (johnxnguyen) [reply app/build.gradle]: Fixed.

**Pull request merged**